.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Everything under this `steps:` here needs to be indented two more spaces. Also, I like the pattern of having a blank line between each step, as you did in the CI workflow.

Reply:
Fixed step indentation issue in the release workflow